moped-database/docker-compose.yml

```diff
@@ -24,6 +24,24 @@ services:
         command: ["graphql-engine", "serve",  "--enable-console"]
 
 
+    moped-pgsql:
+        image: "frankinaustin/postgis:12-3.4"
+        volumes:
+            - moped-db-vol:/var/lib/postgresql/data
+        expose:
+            - 5432
+        ports:
+            - 5432:5432
+        environment:
+            - POSTGRES_USER=moped
+            - POSTGRES_PASSWORD=moped
+            - POSTGRES_DB=moped
+        command: ["postgres",
+            "-c", "jit=off",
+            "-c","log_statement=ddl",
+            "-c", "log_min_messages=info"]
+
+
```

Comment:
I'm able to pull in the service that was previously defined in the platform specific docker compose files into the main file. This is possible now because I've built a single, multi-arch image that exactly matches the versions we have in production, where as previously, we had a pair of single architecture images, and had to pick the right one based on the chip in the computer running it.